src/Sylius/Bundle/AdminBundle/Templating/Helper/AvailableLocaleHelper.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AdminBundle\Templating\Helper;
+
+use Sylius\Component\Resource\Translation\Provider\TranslationLocaleProviderInterface;
+use Symfony\Component\Templating\Helper\Helper;
+
+class AvailableLocaleHelper extends Helper
```

Comment:
Could be `final`

Reply:
i can't do it because behats don't allow this class final :(
![Screenshot 2019-04-26 at 12 04 24](https://user-images.githubusercontent.com/29897151/56800563-9aadd500-681b-11e9-97a6-655599a7e8cf.png)